libs/utils/utils.js

```diff
@@ -335,13 +335,24 @@ export async function loadBlock(block) {
 
 export function decorateSVG(a) {
   const { textContent, href } = a;
-  const ext = textContent?.substr(textContent.lastIndexOf('.') + 1);
+  const altTextFlagIndex =  textContent.indexOf('|');
+  const sanitizeTextContent = altTextFlagIndex === -1
+    ? textContent
+    : textContent?.slice(0 ,altTextFlagIndex).trim();
```

Comment:
Minor spacing issue. Is your eslint working?
```suggestion
    : textContent?.slice(0, altTextFlagIndex).trim();
```

Reply:
>Is your eslint working?

I thought it was. Looks like I need to double check.